index.html

```diff
@@ -48,55 +43,157 @@ <h1>My Name</h1>
       </nav>
     </header>
     <main tabindex="0">
-      <section id="about">
-        <header><h2>About Me</h2></header>
-        <p>
-          Lorem, ipsum dolor sit amet consectetur adipisicing elit. Nulla nisi
-          excepturi quidem, eum rem doloribus quam aspernatur hic enim eligendi
-          commodi minima ullam necessitatibus, cumque blanditiis, nihil magni
-          amet consectetur?
-        </p>
-      </section>
-      <section id="projects">
-        <header><h2>Projects Showcase</h2></header>
-        <p>
-          Lorem, ipsum dolor sit amet consectetur adipisicing elit. Nulla nisi
-          excepturi quidem, eum rem doloribus quam aspernatur hic enim eligendi
-          commodi minima ullam necessitatibus, cumque blanditiis, nihil magni
-          amet consectetur?
-        </p>
-      </section>
+      <div class="main-content">
+        <section id="about">
+          <header><h2>About Me</h2></header>
+          <p>
+            I'm Rabia, a Manchester resident since 2017 and a proud mother of
+            two. My background is rooted in architecture, where I developed a
+            passion for design and precision.
+          </p>
+          <p>
+            After years in the field, I've embarked on a new journey as a
+            full-stack developer student at CODE YOUR FUTURE. I'm immersed in
+            the world of HTML, CSS, JavaScript, React, Node.js, and more.
+          </p>
+          <p>
+            My primary drive is to build a stellar portfolio and secure a
+            meaningful role in the tech industry. Each line of code I write
+            fuels my ambition to contribute to innovative solutions. I believe
+            in the endless possibilities that the tech landscape offers and am
+            committed to continuous growth and learning.
+          </p>
+          <p>
+            My family's support and the CODE YOUR FUTURE community inspire me on
+            this exciting path. I'm eager to explore how I can make a lasting
+            impact on the ever-evolving tech world.
```

Comment:
I love the way this is written. Nice to hear about your background Only thing I would say is CYF is written Code Your Future not CODE YOUR FUTURE